src/Directory.Build.props

```diff
@@ -19,4 +19,10 @@
 
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- Includes source files needed to be compiled into every project. -->
+    <!-- (Based on the idea presented at https://stackoverflow.com/a/70984352/8656352) -->
+    <Compile Include="$(MSBuildThisFileDirectory)Shared\**\*.cs" Link="%(RecursiveDir)%(Filename)%(Extension)"></Compile>
```

Comment:
this is probably redundant, only Esprima needs this now and works without too

Reply:
Yeah, it's kind of future-proofing for the case of refactoring the JSX stuff into a separate project. But if you say so, I'll change it to simply include that file in the Esprima project for now.

Reply:
A bit of "magic" for singl, simple, file that could be copied to new projects, but fine as-is too.

Reply:
I'm convinced. I'll leave the Shared directory as is but will add that file as link to the Esprima project file. This way we can keep it explicit but also future-proof.